src/CommunityVoices/Public/js/quote-form.js

```diff
@@ -12,3 +12,14 @@ function clickEdited() {
     originalText.value = '';
   }
 }
+
+function displayCount(box,count) {
+  var countDisplay = document.getElementById(count);
```

Comment:
Please use `const` here.  I can explain why sometime if you'd like, but `var` is somewhat evil.

Reply:
I addressed your JS comments in new commit.